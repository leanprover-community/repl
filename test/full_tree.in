{"cmd": "theorem f : 1 + 1 = 2 := rfl", "conclusions": true, "infotree": "full"}

{"cmd": "example (P : Prop) (hp : P) : P := by\n  exact hp\n", "conclusions": true, "infotree": "full"}

{"cmd": "variable (a : Prop)\ntheorem f (b : Prop) (hb : b) : Or a b := by\nright\nexact hb", "infotree": "full", "conclusions": true}
