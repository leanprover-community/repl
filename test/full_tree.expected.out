{"env": 0,
 "conclusions":
 [{"pp": "1 + 1 = 2",
   "pos": {"line": 1, "column": 12},
   "endPos": {"line": 1, "column": 21}}]}

